api/SVGGeometryElement.json

```diff
@@ -11,13 +11,13 @@
             "version_added": "18"
           },
           "edge": {
-            "version_added": "≤79"
+            "version_added": "12"
```

Comment:
In order to satisfy the lint it's necessary to say that `SVGGeometryElement` was introduced when some of its members were first introduced on `SVGPathElement`. I think this is the right thing to do and support for the interface itself isn't very interesting, but should I use `partial_implementation` for the range where the interface actually didn't exist?

Reply:
Yes, I think this is the right thing to do. I think it's nice to have the symmetry between the `partial_implementation` for the subfeatures and the parent. It's a little messy, but the reality is messy too.

Reply:
OK, I'm going to split and use notes like the one on `BaseAudioContext`.

Reply:
Gah, I shouldn't have look at that! It took https://github.com/mdn/browser-compat-data/pull/9513 + https://github.com/mdn/browser-compat-data/pull/9514 + https://github.com/mdn/browser-compat-data/pull/9515 + https://github.com/mdn/browser-compat-data/pull/9516 to arrive at something to copy.